frontend/view-edit-client/client-home/view-edit-basic-info.component.tsx

```diff
@@ -81,12 +81,14 @@ export default function ViewEditBasicInfo(props: ViewEditBasicInfoProps) {
               </tr>
             </tbody>
           </table>
-          <button
-            className="secondary edit-button"
-            onClick={() => setEditing(true)}
-          >
-            Edit
-          </button>
+          {props.editable && (
```

Comment:
When adding a client, we don't want them to be able to edit the client on the confirm page. They must press the back button to go edit it. Doing this saves a substantial amount of development work to get the `update` functions all implemented.